.github/workflows/codeql.yml

```diff
@@ -90,7 +90,6 @@ jobs:
           l_max_size: '1000'
           xl_label: 'size/xl'
           fail_if_xl: 'false'
-          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are you sure? The labels def require a token. I believe it must be passed explicitly.